tic_tac_toe/game.py

```diff
@@ -63,22 +92,53 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
+    rows = (
+        ((0, 0), (0, 1), (0, 2)),
+        ((1, 0), (1, 1), (1, 2)),
+        ((2, 0), (2, 1), (2, 2))
+    )
+
+    columns = (
+        ((0, 0), (1, 0), (2, 0)),
+        ((0, 1), (1, 1), (2, 1)),
+        ((0, 2), (1, 2), (2, 2))
+    )
+
+    diagonals = (
+        ((0, 0), (1, 1), (2, 2)),
+        ((0, 2), (1, 1), (2, 0))
+    )
+
+    winning_combinations = rows + columns + diagonals
```

Comment:
_nice!_ we conjectured we could do a for loop for rows and columns but were too tired to implement it. thanks!

@ammaraziz fyi